### DISCUSSION TITLE
is there any template/example for plugin development?

### DISCUSSION THREAD

**Discussion by nilooy** [react-figma/react-figma#559] [Q&A] [answered]
I'm trying to develop a figma plugin with react-figma as renderer and a react app for the main html pop up ui,
Does any one knows any example where this kind of approach has been put together?

**Comment by ilyalesik** [comment]
I think it's technically possible, but there are no example. An issue: https://github.com/react-figma/react-figma/issues/475

**Comment by ilyalesik** [comment, marked as answer]
https://github.com/react-figma/react-figma/tree/master/examples/ui
